templates/emails/user.yearlyreport.hbs

```diff
@@ -3,77 +3,87 @@ Subject: 🏅Your year on Open Collective
 {{> header}}
 
 <style>
-.host {
-  margin: 20px 0px;
-}
-th {
-  padding: 10px;
-  font-weight: normal;
-  font-size: 14px;
-  vertical-align: top;
-}
-th a {
-  color: #7A7B82;
-  text-decoration: none;
-}
-.host .CollectiveCard {
-  margin: 0px 6px 16px 2px;
-}
-.stats { 
-  border-radius: 30px;
-  max-width: 300px;
-  margin: 10px auto;
-  box-shadow: 0 1px 3px 0 rgba(45, 77, 97, 0.1);
-  border: 1px solid rgba(45, 77, 97, 0.1);
-  padding: 10px;
-  text-align: center;
-}
-.stats .btn {
-  width: 100%;
-  background: #80d823;
-  border: 1px solid transparent;
-  border-radius: 28px;
-  display: inline-block;
-  padding: .75rem 1rem;
-  text-transform: uppercase;
-  text-align: center;
-  font-weight: 700;
-  min-width: 150px;
-  box-sizing: border-box;
-  vertical-align: middle;
-  color: #fff;
-  text-decoration: none;
-  margin-bottom: 5px;
-}
-
-.largeEmoji {
-  margin: 15px;
-  font-size: 64px;
-}
+  .host {
+    margin: 20px 0px;
+  }
+
+  th {
+    padding: 10px;
+    font-weight: normal;
+    font-size: 14px;
+    vertical-align: top;
+  }
+
+  th a {
+    color: #7A7B82;
+    text-decoration: none;
+  }
+
+  .host .CollectiveCard {
+    margin: 0px 6px 16px 2px;
+  }
+
+  .stats {
+    border-radius: 30px;
+    max-width: 300px;
+    margin: 10px auto;
+    box-shadow: 0 1px 3px 0 rgba(45, 77, 97, 0.1);
+    border: 1px solid rgba(45, 77, 97, 0.1);
+    padding: 10px;
+    text-align: center;
+  }
+
+  .stats .btn {
+    width: 100%;
+    background: #80d823;
+    border: 1px solid transparent;
+    border-radius: 28px;
+    display: inline-block;
+    padding: .75rem 1rem;
+    text-transform: uppercase;
+    text-align: center;
+    font-weight: 700;
+    min-width: 150px;
+    box-sizing: border-box;
+    vertical-align: middle;
+    color: #fff;
+    text-decoration: none;
+    margin-bottom: 5px;
+  }
+
   .btn {
     height: auto;
   }
+
+  .largeEmoji {
+    margin: 15px;
+    font-size: 64px;
+  }
 </style>
 
 
 <div class="stats">
   <div class="largeEmoji">🎁</div>
   <p>{{tweet.text}}<br />
-  <a href="https://opencollective.com/{{collective.slug}}?{{utm}}">https://opencollective.com/{{collective.slug}}</a></p>
```

Comment:
Can we start removing `utm` parameters?

Reply:
yes good idea, it was for back in the days when we had for a short time Google analytics.
But wouldn't this utm tags be understood by the other open source analytics service that you wanted to use? In this case we should keep it no?

Reply:
Ok, let's not do that until we have more visibility then.